src/main/fc/settings.c

```diff
@@ -741,6 +742,11 @@ const clivalue_t valueTable[] = {
     { "vcd_v_offset",               VAR_INT8    | MASTER_VALUE, .config.minmax = { -15, 16 }, PG_VCD_CONFIG, offsetof(vcdProfile_t, v_offset) },
 #endif
 
+// PG_MAX7456_CONFIG
+#ifdef USE_MAX7456
+    { "max7456_clock",              VAR_UINT8| MASTER_VALUE,    .config.minmax = { 0, 2 }, PG_MAX7456_CONFIG, offsetof(max7456Config_t, clockConfig) },
```

Comment:
This could do with a lookup.

Reply:
I was thinking about `HALF`, `DEFAULT` and `FULL`.
May be `OCVARIABLE` or `OCVAR` instead of `DEFAULT`. 

Thoughts?

Reply:
`HALF`, `DEFAULT`, and `FULL` sounds right to me, and lets the user know what the default is. :-)